client/question/domain/FeedbackDetailsObjectFactory.js

```diff
@@ -275,6 +289,22 @@ tie.factory('FeedbackDetailsObjectFactory', [
       return this._feedbackCategory;
     };
 
+    /**
+     * A getter for the _errorLineNumber property.
+     * This function should return a number representing the line number
+     * where an error in the student code was detected.
+     *
+     * @returns {string}
+     */
+    FeedbackDetails.prototype.getErrorLineNumber = function() {
+      if (
+          this._feedbackCategory !== FEEDBACK_CATEGORIES.SYNTAX_ERROR &&
+          this._feedbackCategory !== FEEDBACK_CATEGORIES.RUNTIME_ERROR) {
+        throw Error('Non-syntax or runtime errors have no error string.');
```

Comment:
Also add a test for this case -- it seems like it was added and I don't see a corresponding test.

Reply:
Done